DS/DSAssignment2/Node.java

```diff
@@ -0,0 +1,232 @@
+public class Node<T> {
+	
+	private T nodeValue;
+	private Node<T> start;
+	private Node<T> next;
+
+	public Node() {
+		nodeValue = null;
+		next = null;
+	}
+     
+	
+	
+	public T getNodeValue() {
+		return nodeValue;
+	}
+
+
+
+	public void setNodeValue(T nodeValue) {
+		this.nodeValue = nodeValue;
+	}
+
+
+
+	public Node<T> getStart() {
+		return start;
+	}
+
+
+
+	public void setStart(Node<T> start) {
+		this.start = start;
+	}
+
+
+
+	public Node<T> getNext() {
+		return next;
+	}
+
+
+
+	public void setNext(Node<T> next) {
+		this.next = next;
+	}
+
+
+
+	public Node(T item) {
+		nodeValue = item;
+		next = null;
+	}
+
+	boolean add(T ele) {
```

Comment:
ACTION: Isn't this method (and many others) should be a part of 'LinkedList' class?